data_steward/resources.py

```diff
@@ -237,6 +259,10 @@ def cdm_schemas(include_achilles=False, include_vocabulary=False):
                     include_table = False
                 elif is_extension_table(table_name):
                     include_table = False
+                elif is_deid_table(table_name):
```

Comment:
For my ticket DC-1001, I added this line to the `cdm_schemas` function:

`elif table_name in FITBIT_TABLES:`
`                     include_table = False`

Are we dong the same thing differently here?

Reply:
It's the same, I just added it in a separate function